client/src/components/Intro/intro.css

```diff
@@ -24,6 +24,10 @@
   border: none;
 }
 
+.intro-description{
```

Comment:
```suggestion
.intro-description {
```

Reply:
@Aanandgupta The linter is expecting the CSS property name to have a space after it and the `{`.

Reply:
Okyy sir will do it